src/olympia/files/tests/test_tasks.py

```diff
@@ -76,6 +78,68 @@ def test_repacking_xpi_files(self):
         # in /tmp.
         assert tmp_dir_before_repacking == sorted(os.listdir(tempfile.gettempdir()))
 
+    @override_switch('enable-manifest-normalization', active=False)
+    def test_does_not_normalize_manifest_json_when_switch_is_inactive(self):
+        upload = self.get_upload('webextension.xpi')
+        fake_results = {'errors': 0}
+        with zipfile.ZipFile(upload.path, 'w') as z:
+            manifest_with_comments = """
+            {
+                // Required
+                "manifest_version": 2,
+                "name": "My Extension",
+                "version": "versionString",
+                // Recommended
+                "description": "haupt\u005fstra\u00dfe"
```

Comment:
I don't think that's a good test. Python will parse those unicode escapes, you gotta double backslash them to keep them as-is in JSON.

Reply:
or use `r"..."` strings